blocks/serializer/index.js

```diff
@@ -0,0 +1,38 @@
+/**
+ * External dependencies
+ */
+import * as query from 'hpq';
+
+/**
+ * Takes a block list and returns the serialized post content
+ *
+ * @param  {Array}  blocks Block list
+ * @return {String}        The post content
+ */
+export default function serialize( blocks ) {
+	return blocks.reduce( ( memo, block ) => {
+		const blockType = block.blockType;
+		const blockSettings = wp.blocks.getBlockSettings( blockType );
+		const rawContent = wp.element.renderToString(
+			blockSettings.save( block.attributes )
```

Comment:
I think we could rename it `serialize` and return the string directly in the function.
To avoid having to call `renderToString` and to avoid any extra React attributes. @aduth may disagree here?